be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -51,7 +51,8 @@ OLAPStatus AlphaRowset::do_load(bool use_cache) {
     return OLAP_SUCCESS;
 }
 
-OLAPStatus AlphaRowset::do_create_reader(std::shared_ptr<RowsetReader>* result) {
+OLAPStatus AlphaRowset::create_reader(std::shared_ptr<RowsetReader>* result) {
+    MutexLock release_lock(&_lock);
```

Comment:
why lock? I think you should call load to make sure rowset is loaded?

Reply:
yes, there should be no lock here, I will remove it.
load is called in init()